### PR TITLE
Imriea/feature/jsoup

### PR DESCRIPTION
**Pull request by awi100**
hmu

### CODE REVIEW

**Comment by ethan-rambacher** [comment]
Please do not commit the .idea folder.

File: src/main/java/Crawler.java [hunk 1]
```diff
@@ -1,10 +1,70 @@
...
+    // crawled, and recrawl time.
+  }
+
+  /**
+   * Function to facilitate pulling info from Document Data Store
+   *
+   * @param req Request that makes a GET request to DDS
+   * @return a Response that is what we receive from DDS
+   */
+  private Request pullFromDDS(Request req) {
```
Comment (jjtorborg) [inline src/main/java/Crawler.java]: This method should be in controller not crawler

File: src/main/java/Crawler.java [hunk 2]
```diff
@@ -1,8 +1,68 @@
...
+     */
+    private void pushToDDS(Response res) {
+        // If a URL is crawled that contains a doc that should not be a search result,
+        // make a DELETE request to DDS which includes the crawled URL
+
+        // If the crawled URL contains a doc that could be a search result,
+        // make a PUT request to DDS containing the URL, full html document, out-links (anchors), time
+        // crawled, and recrawl time.
+    }
+=======
```
Comment (ethan-rambacher) [inline src/main/java/Crawler.java]: Remove this merge conflict remnant

File: src/main/java/Crawler.java [hunk 3]
```diff
@@ -12,5 +72,44 @@ public void crawlUrl(String url) {
...
+   */
+  private void pushToDDS(Response res) {
+    // If a URL is crawled that contains a doc that should not be a search result,
+    // make a DELETE request to DDS which includes the crawled URL
+
+    // If the crawled URL contains a doc that could be a search result,
+    // make a PUT request to DDS containing the URL, full html document, out-links (anchors), time
+    // crawled, and recrawl time.
+  }
+>>>>>>> baf0f8f9c10750ee31afa19099ab5b9b17c85938
```
Comment (jjtorborg) [inline src/main/java/Crawler.java]: Please remove the merge conflict tags

File: src/main/java/Crawler.java (hunk 2 above)
Comment (jjtorborg) [inline src/main/java/Crawler.java]: Please remove the merge conflict tags

File: src/main/java/Crawler.java [hunk 4]
```diff
@@ -1,8 +1,68 @@
...
+import org.jsoup.select.Elements;
+
+import java.util.*;
+
+import java.util.List;
+
 public class Crawler {
 
-  public Crawler() {}
+<<<<<<< HEAD
```
Comment (jjtorborg) [inline src/main/java/Crawler.java]: Please remove the merge conflict tags

**Pull request merged**